win_docker_daemon/README.md

```diff
@@ -0,0 +1,99 @@
+# Agent Check: win_docker_daemon
+
+## Overview
+
+This check monitors [win_docker_daemon][1] through the Datadog Agent.  This integration targets gathering docker stats from the daemon api for Windows containers.  The core docker integration uses the filesystem\cgroups to gather performance stats.  This approach does not work with Windows.  This integration attempts to create the same metrics as the docker integration.  It currently targets:
+- events
+- cpu
+- memory
+- network
+
+This module borrows heavily from the old docker_daemon plugin.
+
+### Installation
+
+To install the win_docker_daemon check on your host:
+
+1. Install the [developer toolkit](https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit) on any machine.
+2. Run `ddev release build win_docker_daemon` to build the package.
+3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+4. Upload the build artifact to any host with an Agent andrun `datadog-agent integration install -w path/to/win_docker_daemon/dist/<ARTIFACT_NAME>.whl`.
+
+### Configuration
+
+1. Edit the `win_docker_daemon.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your win_docker_daemon performance data. See the [sample win_docker_daemon.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `win_docker_daemon` under the Checks section.
+
+## Data Collected
+
+### Metrics
+| name | type  | description |
```

Comment:
You don't need to inline the metrics in the README here,  just adding:

```
See [metadata.csv](<LINK_TO_METADATA.CSV_FILE>) for a full list of provided metrics by this integration.
```

will inline automatically the metrics table in the public Datadog documentation.